src/js/chartAuthoring.js

```diff
@@ -26,18 +26,56 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             },
             dataEntryPanel: {
                 type: "floe.chartAuthoring.dataEntryPanel",
-                createOnEvent: "onTemplatesLoaded",
-                container: "{chartAuthoring}.container",
+                createOnEvent: "onPieChartReady",
```

Comment:
I found that independently instantiating them caused this error related to the modelRelay configuration: 

> infusion-custom.js:12039 Uncaught Error: Assertion failure - check console for more details: Error in modelRelay record member "target": {floe.chartAuthoring.pieChart}.model.dataSet must be a reference to an existing component

I found this was also true if the modelRelay was configured in the pieChart component instead of the dataEntryPanel.